src/bin/cratesfyi.rs

```diff
@@ -270,6 +271,17 @@ enum QueueSubcommand {
         #[arg(long, conflicts_with("reference"))]
         head: bool,
     },
+
+    /// Queue rebuilds for broken nightly versions of rustdoc
+    RebuildBrokenNightly {
```

Comment:
I see the behaviour is that when I only give a start date, it's not treated as start, but we only rebuild one nightly. 

I feel like this is actually a good default, 
but it could be cool if we could make that information part of the docs, perhaps even as help-string for the arg?

Reply:
wow, just realized that `clap` is actually already using the docstring as help-text :D